rest-api/dao/participant_summary_dao.py

```diff
@@ -379,6 +381,117 @@ def to_client_json(self, model):
 
     return result
 
+  def make_csv(self, results):
+    csv_data = StringIO.StringIO()
+    writer = unicode_csv.UnicodeWriter(csv_data)
+    headers = ps_full_data_headers
+    writer.writerow(headers)
+    for row in results.items:
+      line = self.to_client_csv(row)
+      writer.writerow(line)
+    return csv_data.getvalue()
+
+  def to_client_csv(self, row):
+    """this method must append values in the same order as CODE_CONSTANTS.PS_FULL_HEADERS"""
+    line = []
+    if (row.withdrawalStatus == WithdrawalStatus.NO_USE and
+      row.withdrawalTime < clock.CLOCK.now() - WITHDRAWN_PARTICIPANT_VISIBILITY_TIME):
+      row = {k: row.get(k) for k in WITHDRAWN_PARTICIPANT_FIELDS}
+
+    line.append(to_client_participant_id(row.participantId))
+    line.append(to_client_biobank_id(row.biobankId))
+    line.append(row.lastName)
+    line.append(row.firstName)
+    line.append(row.dateOfBirth)
+    if row.languageId:
+      language = self.code_dao.get(row.languageId).value
+      line.append(language)
+    else:
+      line.append(UNSET)
+    line.append(row.enrollmentStatus)
+    line.append(row.consentForStudyEnrollment)
+    line.append(row.consentForStudyEnrollmentTime)
+    line.append(row.consentForElectronicHealthRecords)
+    line.append(row.consentForElectronicHealthRecordsTime)
+    line.append(row.consentForCABoR)
+    line.append(row.consentForCABoRTime)
+    line.append(row.withdrawalStatus)
+    line.append(row.withdrawalTime)
+    line.append(row.streetAddress)
+    line.append(row.city)
+    if row.stateId:
+      state = self.code_dao.get(row.stateId).value
+      line.append(state)
+    else:
+      line.append(UNSET)
+    line.append(row.zipCode)
+    line.append(row.email)
+    line.append(row.phoneNumber)
+    if row.sexId:
+      sex = self.code_dao.get(row.sexId).value
+      line.append(sex)
+    else:
+      line.append(UNSET)
+    if row.genderIdentityId:
+      gender = self.code_dao.get(row.genderIdentityId).value
+      line.append(gender)
+    else:
+      line.append(UNSET)
+    line.append(row.race)
+    if row.educationId:
+      education = self.code_dao.get(row.educationId).value
+      line.append(education)
+    else:
+      line.append(UNSET)
+    if row.numCompletedBaselinePPIModules == 3:
+      line.append(1)
+    else:
+      line.append(0)
+    line.append(row.numCompletedPPIModules)
+    line.append(row.questionnaireOnTheBasics)
+    line.append(row.questionnaireOnTheBasicsTime)
```

Comment:
What format do all these timestamps get returned as? You may want to be explicit about formatting them... e.g.:

https://github.com/all-of-us/raw-data-repository/blob/master/rest-api/dao/participant_summary_dao.py#L367